src/elements/autocomplete/autocomplete-base-element.scss

```diff
@@ -19,6 +19,10 @@
   display: none;
 }
 
+:host([size='s']) {
```

Comment:
does it also affect the select and therefore should be placed in the panel mixins? or more general, I think currently select doesn't sync size s to it's panel options, should it? @mcilurzo Should the select options also adapt its size to the select (currently I don't see it in Figma).